store/pg_engine.go

```diff
@@ -67,24 +67,28 @@ type DB struct {
 	// If true, database will be opened in readonly mode
 	readonly bool
 
-	// Bytebase release version
-	releaseVersion string
+	// Bytebase server release version
+	serverVersion string
+
+	// schemaVersion is the version of Bytebase schema.
+	schemaVersion int
 
 	// Returns the current time. Defaults to time.Now().
 	// Can be mocked for tests.
 	Now func() time.Time
 }
 
 // NewDB returns a new instance of DB associated with the given datasource name.
-func NewDB(logger *zap.Logger, dsn string, connCfg dbdriver.ConnectionConfig, seedDir string, forceResetSeed bool, readonly bool, releaseVersion string) *DB {
+func NewDB(logger *zap.Logger, dsn string, connCfg dbdriver.ConnectionConfig, seedDir string, forceResetSeed bool, readonly bool, schemaVersion int, serverVersion string) *DB {
```

Comment:
nit: Keep the same order as the declaration

Reply:
Done.